### ISSUE TITLE
classDC rank gets set to untrained for level 1 character

### ISSUE THREAD

**Issue by zajosk** [MrPrimate/pathmuncher#13]
The imported level 1 character gets its classdc proficiency set to 0 untrained, which makes the classDC lower than intended 

Steps to reproduce:
1. create new level 1 character in pathbuilder, export json
2. create fresh actor in foundry
3. run pathmuncher with json id from pathbuilder, leave everything default
4. classdc rank is 0. It should be 1 (can be seen in the debug console)

**Expected behavior**
For a level 1 character, classdc rank should be 1 - trained

**Environment:**
 - Foundry Version [10.291]
 - Pathfinder System Version [e.g 4.11.2]
 - Module Version [e.g 0.4.1]
 - Does this behaviour occur with other modules disabled: Yes

 - In Pathbuilder please got to Export -> "Share Copy of Character" and provide the link - https://pathbuilder2e.com/launch.html?build=422753.

**Comment by zajosk** [comment]
I have raised the issue, because I was comparing outputs from pathbuilder with the output from foundry. Pathbuilder was reporting a higher classdc than foundry, that's how I found out. Also I double checked, that the json output from pathbuilder contained the same value for classdc as for other attributes (arcana etc) that were set correctly as trained.

But now that you mentioned it,  https://2e.aonprd.com/ lists the fighter with a trained fighter class dc and the wizard does not have anything like that. So that basically means pathbuilder is wrong and somehow pathmuncher corrects the incorrectly provided data?
